src/web/modules/users/users.show.njk

```diff
@@ -37,11 +35,23 @@
   })
 }}
 
-<h1 class="govuk-heading-m">User services</h1>
-{{ govukSummaryList({
-  rows: format.serviceRoles(payUser, payUser.service_roles)
-  })
-}}
+<dl class="govuk-summary-list">
+  {% for role in payUser.service_roles %}
+  <div class="govuk-summary-list__row">
+    <dd class="govuk-summary-list__value">
+      <a class="govuk-link govuk-link--no-visited-state" href="/services/{{ role.service.external_id }}">
+        {{ role.service.name }}
+      </a>
+    </dd>
+    <dd class="govuk-summary-list__value">
+      <code title="{{ role.service.external_id }}">{{ role.service.external_id | truncate(10) }}</code>
+    </dd>
+    <dd class="govuk-summary-list__actions">
+      <a class="govuk-link govuk-link--no-visited-state" href="/users/{{ payUser.external_id }}/service/{{ role.service.external_id }}/delete">Remove user</a>
```

Comment:
It won't work anyway, so what's the point of adding it now?
https://github.com/alphagov/pay-toolbox/blob/master/src/lib/pay-request/api_utils/adminUsers.js#L67-L73